libs/partners/openai/langchain_openai/chat_models/base.py

```diff
@@ -1401,7 +1412,7 @@ class AnswerWithJustification(BaseModel):
                 )
             tool_name = convert_to_openai_tool(schema)["function"]["name"]
             llm = self.bind_tools(
-                [schema],
+                [schema, *tools] if tools else [schema],
```

Comment:
open question whether we want to support custom ordering here too because that can affect model output

Reply:
allowing passing in schema included

Reply:
It might be needed for experimental reasons, but I'm wondering how we can make the parameters neater.

Option 1:
Check tool names to see whether the schema is included in the tools.

Option 2:
Add a schema_name option to use as the tool_name.

Option 3:
Add the feature when someone raises an issue.